<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Create_Page</class>
 <widget class="QDialog" name="Create_Page">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>380</y>
     <width>181</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Sélectionner une image</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_2">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>200</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Type :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>261</width>
     <height>241</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Retour</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>230</y>
     <width>194</width>
     <height>61</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_type2">
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plante</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Feu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Électrik</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Insecte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Roche</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Psy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poison</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ténèbres</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Acier</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Combat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Glace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dragon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fée</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_type1">
      <item>
       <property name="text">
        <string>Normal</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Plante</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Feu</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Eau</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Électrik</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Vol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Insecte</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Roche</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Sol</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Psy</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Poison</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Spectre</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Ténèbres</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Acier</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Combat</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Glace</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dragon</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Fée</string>
       </property>
      </item>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>30</y>
     <width>381</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="maximumSize">
       <size>
        <width>141</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Nom du Pokémon :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_name"/>
    </item>
   </layout>
  </widget>
  <widget class="QTextBrowser" name="textBrowser_3">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Ubuntu'; font-size:11pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Description :&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QPlainTextEdit" name="plainTextEdit_description">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>60</y>
     <width>301</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>420</y>
     <width>121</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Ajouter un cri</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>420</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Lecture</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_validate">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Valider</string>
   </property>
  </widget>
  <zorder>textBrowser_2</zorder>
  <zorder>label</zorder>
  <zorder>pushButton</zorder>
  <zorder>pushButton_2</zorder>
  <zorder>horizontalLayoutWidget</zorder>
  <zorder>horizontalLayoutWidget_2</zorder>
  <zorder>textBrowser_3</zorder>
  <zorder>plainTextEdit_description</zorder>
  <zorder>pushButton_3</zorder>
  <zorder>pushButton_4</zorder>
  <zorder>pushButton_validate</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
